codacy/values-production.yaml

```diff
@@ -151,6 +151,18 @@ codacy-ingress:
       kubernetes.io/ingress.class: "nginx"
       nginx.ingress.kubernetes.io/use-regex: "true"
 
+## Uncomment these annotations and the `tls:` block to use TLS
+## For guidance on how to set up TLS please refer to the documentation on https://codacy.github.io/chart
+#
```

Comment:
Moved this configuration to values-production because this way there's one less file to edit